crates/telio-lana/src/lib.rs

```diff
@@ -115,3 +132,207 @@ pub fn deinit_lana() -> Result<moose::Result, moose::Error> {
 pub fn is_lana_initialized() -> bool {
     MOOSE_INITIALIZED.load(DEFAULT_ORDERING)
 }
+
+/// Initialize different moose variations (real, file or mock)
+///
+/// # Parameters:
+/// * event_path - path of the DB file where events will be stored. If such file does not exist, it will be created, otherwise reused.
+/// * app_version - Indicates the semantic version of the application.
+/// * prod - whether the events should be sent to production or not
+/// * tx - channel tx half for the InitCallback instance
+pub fn init_moose(
+    event_path: String,
+    app_version: String,
+    prod: bool,
+    tx: SyncSender<bool>,
+) -> std::result::Result<moose::Result, moose::Error> {
+    cfg_if! {
```

Comment:
I don't think we need `cfg_if!` in this case.

Reply:
Leftover... changed.